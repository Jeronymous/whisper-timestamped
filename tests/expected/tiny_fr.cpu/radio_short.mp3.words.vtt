WEBVTT

00:00.060 --> 00:01.660
...

01:00.020 --> 01:09.400
...

01:36.320 --> 01:36.420
...

01:45.020 --> 01:59.000
...

