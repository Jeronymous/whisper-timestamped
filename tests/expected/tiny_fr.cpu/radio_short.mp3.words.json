{
  "text": "............",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.06,
      "end": 1.66,
      "text": "...",
      "tokens": [
        50364,
        1097,
        50614
      ],
      "temperature": 0.0,
      "avg_logprob": -1.4265364408493042,
      "compression_ratio": 0.2727272727272727,
      "no_speech_prob": 0.5112143754959106,
      "words": [
        {
          "text": "...",
          "start": 0.06,
          "end": 1.66
        }
      ]
    },
    {
      "id": 1,
      "seek": 6000,
      "start": 60.02,
      "end": 69.4,
      "text": "...",
      "tokens": [
        50364,
        1097,
        51714
      ],
      "temperature": 0.0,
      "avg_logprob": -0.973953366279602,
      "compression_ratio": 0.2727272727272727,
      "no_speech_prob": 0.8780310153961182,
      "words": [
        {
          "text": "...",
          "start": 60.02,
          "end": 69.4
        }
      ]
    },
    {
      "id": 2,
      "seek": 9000,
      "start": 96.32,
      "end": 96.42,
      "text": "...",
      "tokens": [
        1097
      ],
      "temperature": 0.0,
      "avg_logprob": -0.8283956527709961,
      "compression_ratio": 0.2727272727272727,
      "no_speech_prob": 0.641629695892334,
      "words": [
        {
          "text": "...",
          "start": 96.32,
          "end": 96.42
        }
      ]
    },
    {
      "id": 3,
      "seek": 10500,
      "start": 105.02,
      "end": 119.0,
      "text": "...",
      "tokens": [
        50364,
        1097,
        51714
      ],
      "temperature": 0.0,
      "avg_logprob": -0.728730320930481,
      "compression_ratio": 0.2727272727272727,
      "no_speech_prob": 0.25682491064071655,
      "words": [
        {
          "text": "...",
          "start": 105.02,
          "end": 119.0
        }
      ]
    }
  ],
  "language": "fr"
}